thirdparty/fetch-thirdparty-deps.sh

```diff
@@ -39,10 +39,13 @@ fi
 
 if [ ! -f SharpFont.dll ]; then
 	echo "Fetching SharpFont from nuget"
-	nuget install SharpFont -Version 2.5.0.1
-	cp ./SharpFont.2.5.0.1/lib/net20/SharpFont* .
-	sed '/osx/s@\(dll="\)[^"]*\(" />\)@\1/Library/Frameworks/Mono.framework/Libraries/libfreetype.6.dylib\2@' ./SharpFont.2.5.0.1/Content/SharpFont.dll.config > SharpFont.dll.config
-	rm -rf SharpFont.2.5.0.1
+	nuget install SharpFont -Version 3.0.1
+	cp ./SharpFont.3.0.1/lib/net20/SharpFont* .
+	cp ./SharpFont.3.0.1/config/SharpFont.dll.config .
+	rm -rf SharpFont.3.0.1
+	# No exact version of the dependencies are required on NuGet.
+	# Directory name can change because of version.
```

Comment:
Yes, but we don't want to do version changes by accident like in the middle of a release stabilization so please chose a matching version for us and install it before SharpFont grabs their own.